.jenkinsci/linux-build-wheels.groovy

```diff
@@ -18,7 +19,8 @@ def doPythonWheels() {
 def publishWheels() {
     checkTag = sh(script: 'git describe --tags --exact-match ${GIT_COMMIT}', returnStatus: true)
     withCredentials([usernamePassword(credentialsId: 'ci_nexus', passwordVariable: 'CI_NEXUS_PASSWORD', usernameVariable: 'CI_NEXUS_USERNAME')]) {
-        sh(script: "find wheelhouse -type f -name \"iroha*.whl\" -exec curl -u ${CI_NEXUS_USERNAME}:${CI_NEXUS_PASSWORD} --upload-file {} https://nexus.iroha.tech/repository/artifacts/iroha-python/{} \\;")
+        sh(script: "find wheelhouse -type f -name \"iroha*.whl\" -exec curl -u ${CI_NEXUS_USERNAME}:${CI_NEXUS_PASSWORD} --upload-file {} https://nexus.iroha.tech/repository/artifacts/iroha-python/${repo}/{} \\;")
```

Comment:
`master` folder on Nexus should also contain a wheel with the `latest` suffix. See my previous comment. It will help in locating the latest version of the wheels. Thus, there should be two upload commands if it is a `master` wheel: one that uploads it as usual (you already have it) and another that overwrites the previous `latest` tag.